src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -1190,7 +1193,11 @@ private void parseRepairFrontierRules(final List<Element> elements, final Repair
   }
 
   private void parseAttachments(final Element root) throws GameParseException {
-    for (final Element current : getChildren("attachment", root)) {
+    final ExecutorService service = Executors.newCachedThreadPool();
```

Comment:
Probably want to use a fixed thread pool here.  When I load WW2 Global, this executor service spawns 500+ threads.

Reply:
Yeah I know, I wanted to know how well this executor for "multiple short lived operations" works...

Reply:
Interestingly enough the overall map parsing is faster the more threads are being used, at least on my machine.
When using just 8 Threads, I can barely measure a difference on my machine, while 64 Thread cause a bigger improvement